lab-shawn/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greeting = require('./lib/greet.js');
+const arg = process.argv.slice(2);
+
+greeting.greet(arg);
```

Comment:
Good job in finding out about process.argv and creative take on the implementation by using splice. You could have also used process.argv[2] instead, this represents the third element at the 2nd index in the argv array on the global process object. When you run a file in node, the first element in the argv array is always node and the 2nd element is the <filename> that you're running. For instance, if you want greet to dynamically generate a name you would type: node <filename> Shawn.

Shawn being the at the 2nd index on the argv array, hence prcoess.argv[2]